### PR TITLE
Add repair table command to the Schema Loader Tool

### PR DESCRIPTION
**Pull request by Torch3333**
This adds a `--repair-all` command the Schema Loader Tool used to repair the tables present in the schema file.

For the detailed command usage, please refer to the "Repair table" section in the [readme](https://github.com/scalar-labs/scalardb/blob/d97fe328c9ee460af0db366e899f96bcef861935/schema-loader/README.md#repair-tables).

### CODE REVIEW

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: schema-loader/README.md [hunk 1]
```diff
@@ -80,6 +84,10 @@ Create/Delete Cosmos DB schemas
       --no-scaling       Disable auto-scaling for Cosmos DB
   -p, --password=<key>   Cosmos DB key
   -r, --ru=<ru>          Base resource unit
+      --repair-all       Repair tables : it repairs the table metadata of
```
Comment (feeblefakie) [inline schema-loader/README.md:87]: If we can switch sub-commands for each database, then isn't it less confusing to change the description for each?
In short, talking about `When using Cosmos DB ...` in the dynamo mode looks a bit weird.

**Review by feeblefakie — COMMENTED** [review]
Looking good! Thank you!
Left one minor comment.

File: schema-loader/README.md (hunk 1 above)
Comment (Torch3333) [reply schema-loader/README.md:87]: Right, I agree. Thank you for pointing this out. 
Having a different `--repair-all` command description only for the Cosmos DB one meant I had to do a bit of refactoring. It was revised in https://github.com/scalar-labs/scalardb/pull/622/commits/f7992a0f3af7da3577c5177a5dfa40b3acd74ea2
Please have a look when you can.

File: schema-loader/src/main/java/com/scalar/db/schemaloader/command/CassandraCommand.java [hunk 2]
```diff
@@ -55,6 +56,9 @@ public class CassandraCommand extends StorageSpecificCommand implements Callable
       description = "Cassandra replication factor")
   private String replicationFactor;
 
+  @ArgGroup(exclusive = true)
+  DeleteOrRepairTables deleteOrRepairTables;
```
Comment (brfrn169) [inline schema-loader/src/main/java/com/scalar/db/schemaloader/command/CassandraCommand.java:60]: We can make it `private`?
```suggestion
  private DeleteOrRepairTables deleteOrRepairTables;
```

The same is also true for the other `XXXCommand`?

**Review by brfrn169 — COMMENTED** [review]
Left a comment. Please take a look when you have time!

File: schema-loader/src/main/java/com/scalar/db/schemaloader/command/CassandraCommand.java (hunk 2 above)
Comment (Torch3333) [reply schema-loader/src/main/java/com/scalar/db/schemaloader/command/CassandraCommand.java]: Yes, thank you for pointing this out. It is revised in https://github.com/scalar-labs/scalardb/pull/622/commits/b6b479940291a124586722c1fd9be51074d422c9

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**